pkg/pubsub/adapter/converters/storage.go

```diff
@@ -23,17 +23,18 @@ import (
 	"go.uber.org/zap"
 	"knative.dev/pkg/logging"
 
-	cloudevents "github.com/cloudevents/sdk-go"
-	cepubsub "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub"
+	"github.com/cloudevents/sdk-go"
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub"
+	cepubsub "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub/context"
 )
 
 var (
 	// Mapping of GCS eventTypes to CloudEvent types.
 	storageEventTypes = map[string]string{
-		"OBJECT_FINALIZE":        "google.storage.object.finalize",
-		"OBJECT_ARCHIVE":         "google.storage.object.archive",
-		"OBJECT_DELETE":          "google.storage.object.delete",
-		"OBJECT_METADATA_UPDATE": "google.storage.object.metadataUpdate",
+		"OBJECT_FINALIZE":        Finalize,
```

Comment:
`Finalize` and all other keys need to be storage namedspaced. This is at the converter package level so it will conflict in the future with the next component that has `Finalize`.

Reply:
will do, thanks!